src/mod_pvp_titles.cpp

```diff
@@ -47,92 +29,174 @@ class PVPTitles : public PlayerScript
 			uint32 team = Killer->GetTeamId();
 //			uint32 PlusKill = 14;
 
-			switch (Killer->GetUInt32Value(PLAYER_FIELD_LIFETIME_HONORABLE_KILLS))
+            const uint32 RANK_1	= sConfigMgr->GetIntDefault("PvPTitles.Rank_1", 50);
+            const uint32 RANK_2	= sConfigMgr->GetIntDefault("PvPTitles.Rank_2", 100);
+            const uint32 RANK_3	= sConfigMgr->GetIntDefault("PvPTitles.Rank_3", 500);
+            const uint32 RANK_4	= sConfigMgr->GetIntDefault("PvPTitles.Rank_4", 1000);
+            const uint32 RANK_5	= sConfigMgr->GetIntDefault("PvPTitles.Rank_5", 2000);
+            const uint32 RANK_6	= sConfigMgr->GetIntDefault("PvPTitles.Rank_6", 4000);
+            const uint32 RANK_7	= sConfigMgr->GetIntDefault("PvPTitles.Rank_7", 5000);
+            const uint32 RANK_8	= sConfigMgr->GetIntDefault("PvPTitles.Rank_8", 6000);
+            const uint32 RANK_9	= sConfigMgr->GetIntDefault("PvPTitles.Rank_9", 8000);
+            const uint32 RANK_10 = sConfigMgr->GetIntDefault("PvPTitles.Rank_10", 10000);
+            const uint32 RANK_11 = sConfigMgr->GetIntDefault("PvPTitles.Rank_11", 12500);
+            const uint32 RANK_12 = sConfigMgr->GetIntDefault("PvPTitles.Rank_12", 15000);
+            const uint32 RANK_13 = sConfigMgr->GetIntDefault("PvPTitles.Rank_13", 20000);
+            const uint32 RANK_14 = sConfigMgr->GetIntDefault("PvPTitles.Rank_14", 25000);
+
+			if (Killer->GetUInt32Value(PLAYER_FIELD_LIFETIME_HONORABLE_KILLS) == RANK_1 )
```

Comment:
`Killer->GetUInt32Value(PLAYER_FIELD_LIFETIME_HONORABLE_KILLS)` could be wrapped into a constexpr